### PR TITLE
fix(slider): hide the slider when stats.min=stats.max

### PR DESCRIPTION
**Pull request by maxiloc**
For reference the owner of nouislider is not going to fix it (https://github.com/leongersen/noUiSlider/issues/236#issuecomment-85953802)
So let's just hide the slider for now

### CODE REVIEW

**Comment by vvo** [comment]
We can later on do something _maybe_ better by showing a disabled slider but it requires more thinking. Here the fix will still be better than the current behavior (failing)

File: src/widgets/range-slider/__tests__/range-slider-test.js [hunk 1]
```diff
@@ -94,6 +94,49 @@ describe('rangeSlider()', () => {
     });
   });
 
+  context('with 1 result', () => {
```
Comment (vvo) [inline src/widgets/range-slider/__tests__/range-slider-test.js]: context('when rangestats min === stats max')

**Comment by maxiloc** [comment]
@vvo just rebased

File: src/widgets/range-slider/range-slider.js [hunk 2]
```diff
@@ -112,7 +112,7 @@ function rangeSlider({
         };
       }
 
-      let hasNoRefinements = stats.min === null && stats.max === null;
```
Comment (vvo) [inline src/widgets/range-slider/range-slider.js]: wait shouldn't we add more cases instead of modifying a case?

Comment (vvo) [inline src/widgets/range-slider/range-slider.js]: i.e. shouldn't it be:

```
let hasNoRefinements =
  stats.min === null ||
  stats.max === null ||
  stats.min === stats.max;
```

??

Comment (vvo) [inline src/widgets/range-slider/range-slider.js]: me => :door:

**Pull request merged**